packages/@react-spectrum/utils/src/styleProps.ts

```diff
@@ -206,6 +222,11 @@ export function convertStyleProps<C extends ColorVersion>(props: ViewStyleProps<
 
     let prop = getResponsiveProp(props[key], matchedBreakpoints);
     let value = convert(prop, props.colorVersion);
+
+    if (!value) {
```

Comment:
this check is true for a few too many "falsy" values
https://www.chromatic.com/test?appId=5f0dd5ad2b5fc10022a2e320&id=65dbaab40075a76357e68393

it's gotten rid of a `flexBasis={0}` that it shouldn't have
https://github.com/adobe/react-spectrum/blob/a0f0604e6dc461cc50dd923b494b7ad9b9e5bd0c/packages/%40react-spectrum/dialog/chromatic/Dialog.chromatic.tsx#L490